src/Storage.java

```diff
@@ -0,0 +1,48 @@
+public class Storage<K, V> {
+    private static final int ARRAY_LENGTH = 3;
+    private K[] keysArray;
+    private V[] valuesArray;
+
+    public Storage() {
+        keysArray = (K[]) new Object[ARRAY_LENGTH];
+        valuesArray = (V[]) new Object[ARRAY_LENGTH];
+    }
+
+    public V get(K key) {
+        for (int i = 0; i < keysArray.length; i++) {
+            if (key.equals(keysArray[i])) {
+                return valuesArray[i];
+            }
+        }
+        return null;
+    }
+
+    public void put(K key, V value) {
+        if (key != null && value != null) {
+            for (int i = 0; i < keysArray.length; i++) {
```

Comment:
simplify
better create a variables: private int position = 0;
and increment it after adding element
in this way you don't need use cycle in this method and can write:
keyArray[position] = key
valuesArray[position] = value

Reply:
The loop is nessesary, becouse if I put twice by same key (storage.put(65, "Nick");storage.put(65, "Tom");) put method must rewrite value by this key. The maps do that or I'm mistaken?

Reply:
you are right. Map rewrite values